tests/backup/backup_helper.go

```diff
@@ -2149,6 +2157,167 @@ func VerifyLicenseConsumedCount(ctx context.Context, OrgId string, expectedLicen
 	return err
 }
 
+// DeleteRule deletes backup rule
+func DeleteRule(ruleName string, orgId string, ctx context.Context) error {
+	ruleUid, err := Inst().Backup.GetRuleUid(orgID, ctx, ruleName)
+	if err != nil {
+		return err
+	}
+	deleteRuleReq := &api.RuleDeleteRequest{
+		OrgId: orgId,
+		Name:  ruleName,
+		Uid:   ruleUid,
+	}
+	_, err = Inst().Backup.DeleteRule(ctx, deleteRuleReq)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// CreateRestoreWhileReplacingExistingResources creates restore while replacing existing resources
+func CreateRestoreWhileReplacingExistingResources(restoreName string, backupName string, namespaceMapping map[string]string, clusterName string,
```

Comment:
This is already implemented , CreateRestoreWithReplacePolicy --> this even validates sucess in case of replace existing resources

Reply:
Used CreateRestoreWithReplacePolicy. Thanks!